tas-with-installed-tile/README.md

```diff
@@ -30,15 +30,15 @@ To prepare to run this demo:
   secret and entitlements in a policy branch in this demo:
   - `CONJUR_POLICY`: the policy branch used to configure your TAS tile. This is
     where host identities are automatically added by the service broker.
-  - `CONJUR_ACCOUNT`: the account name for your Conjur / DAP instance.
+  - `CONJUR_ACCOUNT`: the account name for your Conjur instance.
   - `CONJUR_AUTHN_LOGIN`: the identity of a Conjur user who can load a policy
     into the `root` branch.
   - `CONJUR_AUTHN_API_KEY`: the API key of your Conjur user.
-  - `CONJUR_APPLIANCE_URL`: the URL of your Conjur / DAP instance. When using a
+  - `CONJUR_APPLIANCE_URL`: the URL of your Conjur instance. When using a
     high-availability DAP master cluster, this should be set to the URL of the
     master load balancer.
   - `CONJUR_SSL_CERTIFICATE`: the PEM-encoded x509 CA certificate chain for your
-     Conjur / DAP instance, provided as a string. This value may be obtained by
+     Conjur instance, provided as a string. This value may be obtained by
      running the command:
      ```sh-session
      $ openssl s_client -showcerts -servername [DAP_INSTANCE_DNS_NAME] \
```

Comment:
I think we can replace the `DAP_` in this variable name with `CONJUR_`. As far as I can tell it's not actually a variable anywhere, it's just a placeholder.